mappings/mp_efo_impc.sssom.tsv

```diff
@@ -0,0 +1,100 @@
+# comment: "Edit source: https://docs.google.com/spreadsheets/d/13_zoFywKXRFoa6VbqLYpjq1HJrO8zWMOTxZOCZ8s3Lw/edit"
+# creator_id: orcid:0000-0001-8314-2140
+# curie_map:
+#   MP: http://purl.obolibrary.org/obo/MP_
+#   EFO: http://www.ebi.ac.uk/efo/EFO_
+#   owl: http://www.w3.org/2002/07/owl#
+#   rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+#   rdfs: http://www.w3.org/2000/01/rdf-schema#
+#   skos: http://www.w3.org/2004/02/skos/core#
+#   sssom: https://w3id.org/sssom/
+#   orcid: https://orcid.org/
+#   obo: http://purl.obolibrary.org/obo/
+#   semapv: https://w3id.org/semapv/vocab/
+# license: https://creativecommons.org/publicdomain/zero/1.0/
+# mapping_provider: https://www.mousephenotype.org
+# mapping_set_description: 'Some MP-EFO mappings curated by Violeta Munoz-Fuentes for IMPC.'
+# mapping_set_id: https://w3id.org/sssom/commons/mouse-human/mappings/mp_efo_impc.sssom.tsv
+subject_id	subject_label	predicate_id	object_id	object_label	mapping_justification	subject_source	object_source	confidence	mapping_date	author_id
+MP:0000013	abnormal adipose tissue distribution	skos:closeMatch	EFO:0007800	body fat percentage	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000180	abnormal circulating cholesterol level	skos:narrowMatch	EFO:0004574	total cholesterol measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000180	abnormal circulating cholesterol level	skos:narrowMatch	EFO:0004611	low density lipoprotein cholesterol measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000180	abnormal circulating cholesterol level	skos:narrowMatch	EFO:0008592	high density lipoprotein particle size measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000181	abnormal circulating LDL cholesterol level	skos:exactMatch	EFO:0004611	low density lipoprotein cholesterol measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140

```

Comment:
This looks like a `skos:broadMatch ` to me. I do not think it is ever a `skos:exactMatch` when the subject has an `abnormal` qualifier and the object is a more general concept that includes both normal and abnormal cases.
The same comment applies to many other examples in this table.

Reply:
I think it would take around 40 minutes.